libraries/platforms/aws-vcs/execution.mk

```diff
@@ -25,47 +25,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# This Makefile fragment defines the rules that are re-used between cosimulation
-# sub-suites.
-
-# REGRESSION_PATH: The path to the regression folder in BSG F1
-ifndef REGRESSION_PATH
-$(error $(shell echo -e "$(RED)BSG MAKE ERROR: REGRESSION_PATH is not defined$(NC)"))
-endif
-
-# TESTBENCH_PATH: The path to the regression folder in BSG F1
-ifndef TESTBENCH_PATH
-$(error $(shell echo -e "$(RED)BSG MAKE ERROR: TESTBENCH_PATH is not defined$(NC)"))
-endif
-
-# REGRESSION_TESTS: The path to the regression folder in BSG F1
-ifndef REGRESSION_TESTS
-$(error $(shell echo -e "$(RED)BSG MAKE ERROR: REGRESSION_TESTS is not defined$(NC)"))
-endif
-
-# EXEC_PATH: The path to the regression folder in BSG F1
-ifndef EXEC_PATH
-$(error $(shell echo -e "$(RED)BSG MAKE ERROR: EXEC_PATH is not defined$(NC)"))
-endif
-
-# The bsg_manycore_runtime headers are in $(LIBRARIES_PATH) (for cosimulation)
-INCLUDES   += -I$(LIBRARIES_PATH)
-INCLUDES   += -I$(BSG_MACHINE_PATH)
-
-# CSOURCES/HEADERS should probably go in some regression file list.
-CDEFINES   += -DCOSIM -DVCS
-CXXDEFINES += -DCOSIM -DVCS
-CXXFLAGS   += -lstdc++
-
-include $(REGRESSION_PATH)/compilation.mk
-
-# link.mk defines all of the linker targets for building executable
-# binaries. It defines the target <test_name> for each regression
-# test.
-include $(TESTBENCH_PATH)/link.mk
-
-# regression.mk defines the targets regression and <test_name.log>
-include $(REGRESSION_PATH)/regression.mk
+# This Makefile fragment defines the rules that are used for executing
+# applications on HammerBlade Platforms
```

Comment:
I REALLY dislike this file. It's fragile and hard to understand. My goal is to replace it with a python "runner" that runs the tests with the appropriate arguments. However, that was one feature too many for this PR